pkg/platform/baremetal/baremetal.go

```diff
@@ -57,6 +57,7 @@ type config struct {
 	EncryptPodTraffic        bool              `hcl:"encrypt_pod_traffic,optional"`
 	IgnoreX509CNCheck        bool              `hcl:"ignore_x509_cn_check,optional"`
 	ConntrackMaxPerCore      int               `hcl:"conntrack_max_per_core,optional"`
+	CLCSnippets              []string          `hcl:"clc_snippets,optional"`
```

Comment:
I think we should be consistent and have `ControllerCLCSnippets` and `WorkerCLCSnippets`.

Reply:
@invidian For baremetal, the CLC snippets is added is different in Terraform code.

For AWS and Packet: Its added as a list.

In baremetal, its added as a map. 

Key representing the name of the machine. Example:

```
  clc_snippets = {
    "node2" = [
      "${file("./units/hello.yaml")}"
    ]
    "node3" = [
      "${file("./units/world.yaml")}",
      "${file("./units/hello.yaml")}",
    ]
 ```
 
 Reason being, as mentioned in the [docs of Lokomotive-Kubernetes](https://github.com/kinvolk/lokomotive-kubernetes/blob/master/docs/advanced/customization.md#usage):
 ```
 Bare-Metal clusters allow different CoreOS Container Linux snippets
to be used for each node (since hardware may be heterogeneous). 
Populate the optional clc_snippets map variable with any controller
or worker name keys and lists of snippets.
 ```

Reply:
I ma updating the PR to reflect the above nature.